src/Domain/Operation/TransactionCreationData.php

```diff
@@ -7,22 +7,22 @@
 use Money\Money;
 use Bookkeeper\Accounting\Domain\Account\AccountIdInterface;
 
-final class CreateTransactionData
+final class TransactionCreationData
 {
     public TransactionIdInterface $id;
-    public AccountIdInterface $debitAccountId;
     public AccountIdInterface $creditAccountId;
+    public AccountIdInterface $debitAccountId;
     public Money $amount;
 
     public function __construct(
         TransactionIdInterface $id,
-        AccountIdInterface $debitAccountId,
         AccountIdInterface $creditAccountId,
+        AccountIdInterface $debitAccountId,
```

Comment:
Does order (debit/credit or credit/debit) it matted?

Reply:
In natural language we usually say "From point A to point B". This means, resource point (credit) goes first. In my opinion, the same order in function arguments is more intuitive.

Reply:
Ok, for me equal :)